content/docs/user-guide/remote-projects.md

```diff
@@ -0,0 +1,131 @@
+# Working with projects and remote objects
+
+<details>
```

Comment:
This detail block needs to be removed IMO. this page doesn't use the `dvc` branch of the [example repo](https://github.com/iterative/example-mlem-get-started) at all

Reply:
cc: @aguschin why did we consider this in the original content?